### PR TITLE
Update verify ec-task reference

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Review by zregvart — APPROVED** [review]

File: components/enterprise-contract/kustomization.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ configMapGenerator:
 - name: ec-defaults
   namespace: enterprise-contract-service
   literals:
-  - verify_ec_task_bundle=quay.io/hacbs-contract/ec-task-bundle:a98163a138491ba91ffce4e1f1853e3a52943f1e
+  - verify_ec_task_bundle=quay.io/hacbs-contract/ec-task-bundle:741b8a1fd9a3e1019e494d5136000aefde10c590@sha256:eb0cd5e3978c372bc82a5e0ab34d7d4d9c74ef1d18d6ef424ea15d1c0e881d44
```
Comment (simonbaird) [inline components/enterprise-contract/kustomization.yaml:21]: Looking at https://quay.io/repository/hacbs-contract/ec-cli?tab=tags&tag=latest it seems right to me.

Comment (simonbaird) [reply components/enterprise-contract/kustomization.yaml:21]: wait that's not it...

Comment (simonbaird) [reply components/enterprise-contract/kustomization.yaml:21]: https://quay.io/repository/hacbs-contract/ec-task-bundle?tab=tags&tag=latest

**Review by simonbaird — APPROVED** [review]

File: components/enterprise-contract/kustomization.yaml (hunk 1 above)
Comment (lcarva) [reply components/enterprise-contract/kustomization.yaml:21]: Second link looks right to me:
image

**Review by zregvart — APPROVED** [review]

**Pull request merged**